### ISSUE TITLE
CTD when trying to use autoclothing

### ISSUE THREAD

**Issue by TealSeer** [DFHack/dfhack#2700]
crashed to desktop when running the command `autoclothing report`. For additional context I had previously enabled tailor though I don't know if that changes anything, also not really sure if dfhack dumps error logs anywhere but I can attach those if needed and I can attach my save game as well. I'm trying to replicate the issue again but I can't seem to get it to happen again is the problem.

Version: DFHack 50.05-alpha2 for the Steam version

**Comment by ab9rf** [comment]
could you please upload a save (we need the entire "region_n_" folder) to dffd (or your favorite file service, if you don't want to use dffd) and provide a link?

**Comment by TealSeer** [comment]
For sure, [this](https://dffd.bay12games.com/file.php?id=16383) is a manual save I made right before I started testing the commands. Also worth noting that on my second attempt to replicate this while it didn't immediately crash like before, it ended up crashing like 30 minutes later seemingly randomly.

**Comment by Danarca** [comment]
Had the same issue. Used `autoclothing report` on alpha2, CTD.
[last save I could find](https://dffd.bay12games.com/file.php?id=16385)

**Comment by omgpointless** [comment]
Same issue here. I don't have much valuable data to provide other than affirming this affects more users.

CTD on a build from latest develop at the time, compiled 1st feb. 
Was using autoclothing report successfully, during my save I enabled and disabled tailor. I also set a couple of items in autoclothing and later removed them from autoclothing. These "on and off" on tailor and messing with autoclothing happened over the course of saving and reload 3 times during the day. As I was about to call it quits, with tailor and autoclothing both disbaled, and manual orders created instead. I decided to run autoclothing report one last time (I like that you get stats on the status) and after pressing enter DF crashed to desktop.
